### PR TITLE
Update locale interface for form-behaviors

### PR DESCRIPTION
**Pull request by jim-benson**
Task: task-[work-item-number]

Link: preview-[pull-request-number]

I noticed I missed updating the locale interface.

## Contributor checklist

- [ ] Did you update the description of this pull request with a review link and test steps?
- [ ] Did you submit a changeset? Changesets are required for all code changes.
- [ ] Does your pull request implement complex UI logic (js/ts)? Consider adding an integration test to test your user flow.
- [ ] Does your pull request add a new page to the documentation site? Add your new page for automated accessibility testing in /integration/tests/accessibility.

### CODE REVIEW

File: .changeset/purple-kids-wash.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-js': minor
+---
+
+Update loale interface to include added error states
```
Comment (homing1) [inline .changeset/purple-kids-wash.md:5]:
```suggestion

Update LocStrings interface to include added error states.
```

File: .changeset/purple-kids-wash.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-js': minor
```
Comment (homing1) [inline .changeset/purple-kids-wash.md:2]: I think this is more of patch than a minor.

**Review by BenDMyers — APPROVED** [review]

File: .changeset/purple-kids-wash.md (hunk 1 above)
Comment (jim-benson) [reply .changeset/purple-kids-wash.md]: updated

File: .changeset/purple-kids-wash.md (hunk 2 above)
Comment (jim-benson) [reply .changeset/purple-kids-wash.md]: updated

**Review by homing1 — APPROVED** [review]

**Pull request merged**